next.config.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+module.exports = {
+  sassOptions: {
+    includePaths: [path.join(__dirname, 'styles')],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PeteChu/petechu-website/pullRequest?prid=7929896)
```suggestion
    includePaths: [path.join(__dirname, "styles")],
```